protected/humhub/modules/admin/views/group/reassign-all.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
We do not need an extra confirm view here. 
It's better to simply use the link/button attribute "data-confirm".

e.g. https://github.com/humhub/humhub/blob/master/protected/humhub/modules/admin/views/module/list.php#L54

Reply:
Done!